src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -378,13 +378,29 @@ bool program_node::has_padded_dependency() const {
     });
 }
 
+bool program_node::is_fused_dep(size_t dep_idx) const {
+    for (auto fused : get_fused_primitives()) {
+        if (dep_idx >= fused.dep_start_idx) {
```

Comment:
Does it work correctly for dep_idx = 0 and some fused op w/o dependencies? I assume that dep_start_idx will be also 0 in that case.

Reply:
I think we can utilize total_num_deps too.

Reply:
local_desc.dep_start_idx = fused_node.get_dependencies().size();
Initial value of dep_start_idx comes from fused_node's dep size and it can't be 0.